jdi-light-applitools-tests/src/test/java/com/jdi/testng/TestNGListener.java

```diff
@@ -0,0 +1,58 @@
+package com.jdi.testng;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import static java.lang.System.currentTimeMillis;
+
+import java.lang.reflect.Method;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import org.testng.IInvokedMethod;
+import org.testng.IInvokedMethodListener;
+import org.testng.ITestResult;
+import org.testng.annotations.Test;
+
+public class TestNGListener implements IInvokedMethodListener {
+    long timeStart = 0;
+    String testName = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)